fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/makers/AggregationAverageMaker.java

```diff
@@ -85,6 +86,21 @@ protected LogicalMetric makeInner(String metricName, List<String> dependentMetri
         return new LogicalMetric(outerQuery, NO_OP_MAPPER, metricName);
     }
 
+    @Override
+    protected LogicalMetric makeInner(LogicalMetricInfo logicalMetricInfo, List<String> dependentMetrics) {
+        // Get the Metric that is being averaged over
```

Comment:
Would it make sense to proxy the other method into this one by building a default LogicalMetricInfo that is more or less backwards compatible?

Reply:
@michael-mclawhorn Yes! This would be a brilliant idea! 👍